client/src/app/components/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="md-dialog.fullscreen-dialog">
+    <div class="wrapper">
```

Comment:
Question: why is this class called so strangely? and whats the purpose of the 'wrapper' class? What's it wrapping?